app/src/main/java/de/stephanlindauer/criticalmaps/fragments/ChatFragment.java

```diff
@@ -121,6 +121,8 @@ public boolean onTouch(View v, MotionEvent event) {
                     case MotionEvent.ACTION_UP:
                         isScrolling = false;
                         return false;
+                    default:
```

Comment:
Following sonar rule:
squid:SwitchLastCaseIsDefaultCheck - "switch" statements should end with a "default" clause.